src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -463,6 +489,38 @@ describe('non-empty successful affected clusters table', () => {
       });
     });
 
+    VERSION_COMBINATIONS.forEach((vs) => {
```

Comment:
Now that we have 2 tests for filtering, maybe we should move to the pattern in other pages, a filtersConf object and have tests for single filters and combinations of filters.

Reply:
yes, this makes sense. I'll just drop a TODO to not to overwhelm the PR